ui_tests/src/test/resources/com/saucedemo/account/signInNegative.feature

```diff
@@ -0,0 +1,66 @@
+Feature: Sign In
+
+
+  @signIn
+
+  Scenario: Sign in with empty username and password
```

Comment:
@OzodaAripova This is a valid test case but the name of the scenario doesn't match the steps performed.
Sign in with empty username and password > you are setting the empty username with:
I set '' as username on Sign in page
but you are setting the password to:
I set ' ' as password on Sign in page
there's a whitespace in the single quotes, i.e. the password will get the whitespace value (not empty)

Reply:
See line 19, it's set to empty there

Reply:
Fixed